doc.html

```diff
@@ -16,1281 +16,982 @@
-                                        TBD
-                                    </div>
-                                </div>
-                            </div>
+                    <div id="install" class="section_cardbody">
+                        <p class="cardHeader">How to install</p>
+                        <p class="info_text" style="text-align:justify">See the <a
+                                href="start.html">install section</a> for details.</p>
+                    </div>
+                    <div id="default" class="section_cardbody">
+                        <p class="cardHeader">Defaults</p>
+                        <p class="info_text" style="text-align:justify">The benchmark scripts source
+                            the base script (/opt/pbench-agent/base) which sets a bunch of defaults:
+                        </p>
+                        <p class="c_snip">
+                            pbench_run=/var/lib/pbench-agent
+                            <br>pbench_log=/var/lib/pbench-agent/pbench.log
+                            <br>date=`date "+%F_%H:%M:%S"`
+                            <br>hostname=`hostname -s`
+                            <br>results_repo=pbench@pbench.example.com
+                            <br>results_repo_dir=/pbench/public_html/incoming
+                            <br>ssh_opts='-o StrictHostKeyChecking=no'
+                        </p>
+                        <p class="info_text" style="text-align:justify">These are now specified in
+                            the config file /opt/pbench-agent/config/pbench-agent.cfg.</p>
+                    </div>
+                    <div id="availabletool" class="section_cardbody">
+                        <p class="cardHeader">Available tools</p>
+                        <p class="info_text" style="text-align:justify">The configured default set
+                            of tools (what you would get by running pbench-register-tool-set) is:
+                        </p>
+                        <div class="info_text" style="text-align:justify">
+                            <li>
+                                sar, iostat, mpstat, pidstat, proc-vmstat, proc-interrupts, perf
+                            </li>
                         </div>
+                        <p class="info_text" style="text-align:justify">In addition, there are tools
+                            that can be added to the default set with pbench-register-tool:</p>
+                        <div class="info_text" style="text-align:justify">
+                            <li>
+                                blktrace, cpuacct, dm-cache, docker, kvmstat, kvmtrace, lockstat,
+                                numastat, perf, porc-sched_debug, proc-vmstat, qemu-migrate, rabbit,
+                                strace, sysfs, systemtap, tcpdump, turbostat, virsh-migrate, vmstat
+                            </li>
```

Comment:
Not sure if we would keep `<li>` without the `<ul>'

Reply:
fixed